### DISCUSSION TITLE
How to navigate data in Sheet?

### DISCUSSION THREAD

**Discussion by comdiv** [Byron/google-apis-rs#321] [Q&A]
I see that Sheet struct has `data` field. In documentation i see that by default it has all data.
But if i use `hub.spreadsheets.get(MYID)...sheets` - all of them has None data.
The only method i find to read data is `values_get(id, range)` that returns `Option<Vec<Vec<String>>` , in what cases Sheet.data is not empty?

**Comment by Byron** [comment]
I really don't know. I'd love to give some hints at how to debug this, as ideally one would be able to inspect the server response as JSON to see what kind of data is in there exactly, but even that isn't possible anymore.

On MacOS, I'd use `proxyman` to intercept HTTPS traffic and look at it that way, other operating systems should have a way to put a proxy in the middle, too.

This is far from an answer, but all I got.
